crates/bevy_winit/src/lib.rs

```diff
@@ -39,7 +41,9 @@ use winit::{
     event_loop::{ControlFlow, EventLoop, EventLoopBuilder, EventLoopWindowTarget},
 };
 
-#[cfg(target_arch = "wasm32")]
+use crate::accessibility::{AccessKitAdapters, AccessibilityPlugin, WinitActionHandlers};
+
+#[cfg(target_ = "wasm32")]
```

Comment:
is that supposed to be `target_arch`?

Reply:
I suspect it's whatever's on main. I was probably a bit overzealous at managing merge conflicts in that spot. :) Thanks for catching it, I'll get it reconciled tomorrow.

Reply:
```suggestion
#[cfg(target_arch = "wasm32")]
```

Reply:
Thanks!